data/deprecated.json

```diff
@@ -1205,6 +1205,126 @@
     "old": {"parking": "street"},
     "replace": {"parking": "lane"}
   },
+  {
+    "old": {"parking:lane:both": "yes"},
+    "replace": {"parking:both": "lane"}
+  },
+  {
+    "old": {"parking:lane:both": "no"},
+    "replace": {"parking:both": "no"}
+  },
+  {
+    "old": {"parking:lane:both": "no_parking"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_parking"}
+  },
+  {
+    "old": {"parking:lane:both": "no_standing"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_standing"}
+  },
+  {
+    "old": {"parking:lane:both": "no_stopping"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping"}
+  },
+  {
+    "old": {"parking:lane:both": "fire_lane"},
+    "replace": {"parking:both": "no", "parking:both:restriction": "no_stopping", "parking:both:restriction:reason": "fire_lane"}
+  },
+  {
+    "old": {"parking:lane:both": "separate"},
+    "replace": {"parking:both": "separate"}
+  },
+  {
+    "old": {"parking:lane:both": "diagonal"},
+    "replace": {"parking:both": "lane", "parking:both:orientation": "diagonal"}
+  },
+  {
+    "old": {"parking:lane:both": "parallel"},
+    "replace": {"parking:both": "lane", "parking:both:orientation": "parallel"}
+  },
+  {
+    "old": {"parking:lane:both": "perpendicular"},
+    "replace": {"parking:both": "lane", "parking:both:orientation": "perpendicular"}
+  },
```

Comment:
```suggestion
  {
    "old": {"parking:lane:both": "diagonal"},
    "replace": {"parking:both:orientation": "diagonal"}
  },
  {
    "old": {"parking:lane:both": "parallel"},
    "replace": {"parking:both:orientation": "parallel"}
  },
  {
    "old": {"parking:lane:both": "perpendicular"},
    "replace": {"parking:both:orientation": "perpendicular"}
  },
```

Same as with my comment above: I would think we should rather be more conservative when updating the tags that auto-fill data that might not be true. When you add `parking:lane:both= diagonal` to https://osmberlin.github.io/osm-tag-updater/#/manual we only transform the given key but show a yellow info to add the missing key manually. 
Unfortunately, the way iD works is that the orientation field will become visible, but the "parking" field does not.
It would be better to have it show like …
<img width="372" alt="image" src="https://github.com/openstreetmap/id-tagging-schema/assets/111561/583298e6-4a48-49ef-bcc2-ff23e78aa114">

However I tried that but we are missing the capabilities to show fields conditionally depending on other tags.

Since the deprecations happen automatically (as in, the user is not made aware that things change), I would still vote to only update what we are 100% sure about.